### ISSUE TITLE
Feature request: 1.20.1 EnderIO compatibility

### ISSUE THREAD

**Issue by gardenappl** [TheLMiffy1111/JAOPCA#286]
EnderIO is right now in alpha for 1.20.1, might be too early to support it but would be quite nice to have SAG Mill recipes.

**Comment by TheLMiffy1111** [comment]
Waiting on Team-EnderIO/EnderIO#327

**Comment by furtabs** [comment]
Its merged

**Comment by Puyodead1** [comment]
:sob:

**Comment by TheLMiffy1111** [comment]
FYI the recipes are still absent on 1.20.1
